webapp/src/ts/effects/contacts.effects.ts

```diff
@@ -116,50 +121,47 @@ export class ContactsEffects {
     );
   },{ dispatch: false });
 
-  updateSelectedContactSummary = createEffect(() => {
+  receiveSelectedContactTargetDoc = createEffect(() => {
     return this.actions$.pipe(
-      ofType(ContactActionList.receiveSelectedContactReports),
-      withLatestFrom(
-        this.store.pipe(select(Selectors.getSelectedContact))
-      ),
-      exhaustMap(([, selectedContact]) => {
-        const selected: any = { ...selectedContact };
-        this.contactsActions.setContactsLoadingSummary(true);
-        this.tasksForContactService
-          .get(selected)
-          .then((tasks) => {
-            return this.contactsActions.updateSelectedContactsTasks(tasks);
-          });
-        return from(this.contactSummaryService.get(selected.doc, selected.reports, selected.lineage)).pipe(
-          map(summary => {
-            this.contactsActions.setContactsLoadingSummary(false);
-            return this.contactsActions.updateSelectedContactSummary(summary);
+      ofType(ContactActionList.loadSelectedContactTargetDoc),
+      exhaustMap(({ payload: { selected } }) => {
+        return from(this.targetAggregateService.getCurrentTargetDoc(selected)).pipe(
+          map(targetDoc => {
+            const selectedContact = { ...selected, targetDoc };
+            this.contactsActions.loadSelectedContactSummary(selectedContact);
+            return this.contactsActions.receiveSelectedContactTargetDoc(targetDoc);
           }),
           catchError(error => {
-            console.error('Error loading summary', error);
+            console.error('Error loading target doc', error);
             return of(this.globalActions.unsetSelected());
           }),
         );
       })
     );
   },{ dispatch: false });
 
-  receiveSelectedContactTargetDoc = createEffect(() => {
+  updateSelectedContactSummary = createEffect(() => {
     return this.actions$.pipe(
-      ofType(ContactActionList.updateSelectedContactSummary),
-      withLatestFrom(
-        this.store.pipe(select(Selectors.getSelectedContact))
-      ),
-      exhaustMap(([, selectedContact]) => {
-        return from(this.targetAggregateService.getCurrentTargetDoc(selectedContact)).pipe(
-          map(targetDoc => {
-            return this.contactsActions.receiveSelectedContactTargetDoc(targetDoc);
-          }),
-          catchError(error => {
-            console.error('Error loading target doc', error);
-            return of(this.globalActions.unsetSelected());
-          }),
-        );
+      ofType(ContactActionList.loadSelectedContactSummary),
+      exhaustMap(({ payload: { selected } }) => {
+        this.contactsActions.setContactsLoadingSummary(true);
+        this.tasksForContactService
+          .get(selected)
+          .then((tasks) => {
+            return this.contactsActions.updateSelectedContactsTasks(tasks);
+          });
+        return from(this.contactSummaryService
+          .get(selected.doc, selected.reports, selected.lineage, selected.targetDoc))
```

Comment:
this indentation is kinda off. it looks like you're chaining from the `get` but you're instead chaining from the `from`.